### DISCUSSION TITLE
URL link to local file

### DISCUSSION THREAD

**Discussion by hydrospheric** [phase1geo/Minder#327] [Ideas]
It would be amazing to be able to link nodes to local files (e.g. pdfs).
It shouldn't be too hard to recognize a local path as a valid url, should it?

**Comment by phase1geo** [comment]
You should be able to do that already. As long as the file name is prefixed with a '/' character (the file path must be an absolute pathname), it should highlight the path as a clickable link. To view the file, Control-click the link.

**Reply by fbm224** [reply]
I tried adding a link to a file in minder 1.14.0, but it did not work in any file type but jpg. I also tried using the prefix "file:\\".  
I tried with txt, pdf, and xlsx

**Comment by phase1geo** [comment]
@fbm224 Did you try adding file:// prior to the filepath?  That is the correct URI to use for Linux systems.  The path should probably also be an absolute pathname, so your link should read something like "file:///path/to/the/file/name.extension.  This works in the 1.15.2 version of Minder anyways.

**Reply by fbm224** [reply]
Thank you, it works, although it opens a new Dolphin window pointing to the file and then it's opened by the default program. Do you  know if there is a way to avoid this?, 
Using openSuse Tumbleweed Plasma.

**Reply by phase1geo** [reply]
There is no way to avoid this at the current time.  All files are opened in the file browser.

**Reply by fbm224** [reply]
Ok, thank you, that's close enough. BTW thanks to all the devs. I'm finding minder quite useful

**Comment by phase1geo** [comment]
I think it should work with 1.14.0 also.
